Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs

```diff
@@ -1303,31 +1312,38 @@ public void UpdateTrainDerailmentRisk()
                     CarBehind.FrontAirHoseHeightAdjustmentM = frontairhoseheightadjustmentreferenceM - CarBehind.FrontAirHoseHeightAdjustmentM;
                 }
 
-                var rearairhoseangleadjustmentreferenceRad = (float)Math.Cos(CouplerDistanceThisCarM / CarAirHoseLengthM);
-                var frontairhoseangleadjustmentreferenceRad = (float)Math.Cos(CouplerDistanceBehindCarM / CarAirHoseLengthM);
+                // Calculate angle adjustments
+                var rearairhoseangleadjustmentreferenceRad = (float)Math.Asin(CarAirHoseHorizontalLengthM / CarAirHoseLengthM);
```

Comment:
```suggestion
                var rearAirHoseAngleAdjustmentReferenceRad = (float)Math.Asin(CarAirHoseHorizontalLengthM / CarAirHoseLengthM);
```

Reply:
Chris,

Thanks for the feedback.

Blueprint updated as suggested.

Reply:
Hi Peter,

Thanks. Your blueprint is much better - good example for others.

Chris